app/services/payment_provider_customers/create_service.rb

```diff
@@ -10,7 +10,7 @@ def initialize(customer)
 
     def create_or_update(customer_class:, payment_provider_id:, params:, async: true)
       provider_customer = customer_class.find_or_initialize_by(
-        customer_id: customer.id,
+        customer_id: customer.id, # TODO: Why?
```

Comment:
:thinking:

Reply:
`customer_class` will be a `PaymentProviderCustomers::StripeCustomer` so no changes required here. (Appart from a refact maybe... :D )

Reply:
TODO removed 😅